### PR TITLE
Teacher (and student) homepage and unit library UI/UX changes

### LINKED ISSUE #2786: UI/UX Changes to Teacher homepage
Make the following changes to the Teacher/Student homepage + library:
- Rename "Browse WISE Units" tab to "Unit Library" (confused teachers have asked where the library is)
- Remove "completed" count from run list (confusing to students especially when they see a unit they've finished not be labeled completed; also somewhat extraneous as we show total and active counts)
- ~~Display run ID after title in run listing, remove from run menu (easier access to ID for teachers and researchers to refer to when asking for help)~~ Use Access Code to refer to a run instead
- Re-order run menu (based on common usage, related options): move Edit Content below Share and Add to Google

Site_ Teacher Class Schedule (Oct 2020)

- Add menu to unit info dialog for runs: show parent ID plus Copy and Edit options (provides a more accessible way for teachers to copy a run unit - can only be done from authoring tool as of now; adds a second way to edit content)

Site_ Run Unit Info (Oct 2020)

- Fix pagination numbers in unit libraries:
image

### PR DESCRIPTION
**Pull request by breity**
To test, check that the following changes have been made:
- "Browse WISE Units" tab has been renamed to "Unit Library"
- Completed count has been removed from teacher and student Class Schedules
- "Edit Content" has been moved below share options in run menu
- Menu with Copy and Edit (if user has permission) options have been added to unit info dialog for runs
- Pagination numbers have been fixed in unit libraries (used to show "0 - 12 of XX" instead of "1 - 12 of XX", for example)

Closes #2786.

### CODE REVIEW

File: src/main/webapp/site/src/app/modules/library/libraryPaginatorIntl.ts [hunk 1]
```diff
@@ -16,8 +16,8 @@ export class LibraryPaginatorIntl extends MatPaginatorIntl {
       return $localize`0 of ${length}:total:`;
     }
     length = Math.max(length, 0);
```
[the remark this answers is not in the document]
Comment (hirokiterashima) [reply src/main/webapp/site/src/app/modules/library/libraryPaginatorIntl.ts:18]: Also, I think it's save to remove the constructor() block from this class. The super() call happens automatically.

**Review by hirokiterashima — CHANGES_REQUESTED** [review]
The UI/UX changes are nice improvements!

I added some code review questions inline.

Also, when I try to edit the unit via the library project menu component dialog, The dialog still remains overlaid on the authoring tool, like in this image:

Screen Shot 2020-11-05 at 8 56 09 AM

File: src/main/webapp/site/src/app/modules/library/library-project-menu/library-project-menu.component.ts [hunk 2]
```diff
@@ -17,6 +17,9 @@ export class LibraryProjectMenuComponent implements OnInit {
   @Input()
   project: Project;
 
+  @Input()
+  isRun: boolean;
+
```
[the remark this answers is not in the document]
Comment (breity) [reply src/main/webapp/site/src/app/modules/library/library-project-menu/library-project-menu.component.ts:22]: Ah, yep. Need to hide the share option if it's a run. Forgot to include that change in a commit. Just did now.

File: src/main/webapp/site/src/app/modules/library/libraryPaginatorIntl.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (breity) [reply src/main/webapp/site/src/app/modules/library/libraryPaginatorIntl.ts]: Yeah, good catch. Not sure why that case was in there. I don't think length can be less than 0.
I pushed these changes.

**Comment by breity** [comment]
> 
> Also, when I try to edit the unit via the library project menu component dialog, The dialog still remains overlaid on the authoring tool, like in this image:
>

@hiroki Thanks for catching. Should be fixed now.

**Review by hirokiterashima — APPROVED** [review]

**Pull request merged**